unity/Assets/Scripts/Unity/UnityMap.cs

```diff
@@ -115,5 +117,24 @@ public void OnTilePlaced(object sender, TileMapEventArgs<ITile> e)
             }
             newTile.transform.parent = this.transform;
         }
+
+        public void ShowPossibleScoreForCoordinate(Coordinate coordinate)
+        {
+            if(unityStack.Peek() != null)
```

Comment:
Looks good. Suppose we need to adjust this as soon as the Hand is implemented/merged by @denadex however.

Reply:
Yes, I'd make the same suggestion, merge #30 first and then rebase, should be easier to do this way.

Reply:
Your call, at first glance, I'd say, it would be easier to rebase #30 onto #29 . Haven't tried anything though.